generator/internal/rust/templates/storage/client.rs.in.mustache

```diff
@@ -0,0 +1,76 @@
+{{!
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+}}
+// Copyright {{Codec.Storage.Codec.CopyrightYear}} Google LLC
+{{#Codec.Storage.Codec.BoilerPlate}}
+//{{{.}}}
+{{/Codec.Storage.Codec.BoilerPlate}}
+
+impl StorageControl {
+    /// Returns a builder for [StorageControl].
+    ///
+    /// ```no_run
+    /// # tokio_test::block_on(async {
+    /// # use google_cloud_storage::client::StorageControl;
+    /// let client = StorageControl::builder().build().await?;
+    /// # gax::client_builder::Result::<()>::Ok(()) });
+    /// ```
+    pub fn builder() -> ClientBuilder {
+        gax::client_builder::internal::new_builder(client_builder::Factory)
+    }
+{{#Codec.Storage.Services}}
+{{#Methods}}
+
+{{> ../common/client_method_preamble}}
+    pub fn {{Codec.Name}}(&self) -> crate::builder::{{Codec.ServiceNameToSnake}}::{{Codec.BuilderName}}
+    {
+        self.storage.{{Codec.Name}}()
+    }
+{{/Methods}}
+{{/Codec.Storage.Services}}
+{{#Codec.Control.Services}}
+{{#Methods}}
```

Comment:
Oh, I see, these need to be indented too: I believe mustache skips empty blocks as long as the control structures have the same indentation.